package/run.sh

```diff
@@ -21,11 +36,19 @@ fi
 
 env "INSTALL_RKE2_ARTIFACT_PATH=${CATTLE_AGENT_EXECUTION_PWD}" installer.sh
```

Comment:
We should probably set the `INSTALL_RKE2_TAR_PREFIX` when we are calling the installer instead of having the RKE2 install script determine it for us, which will lead to more predictable behavior (or frustrating debugging) if down the line we decide to change the logic in the RKE2 install script. https://github.com/rancher/rke2/blob/master/install.sh#L40-L43